### PR TITLE
Only do an autoupdate after an ongoing transaction.

### PR DESCRIPTION
**Pull request by ostcar**
This needs at least django 1.9

Fixes: #2248

### CODE REVIEW

File: openslides/utils/autoupdate.py [hunk 1]
```diff
@@ -108,14 +109,19 @@ def inform_changed_data(instance, is_deleted=False):
...
-                'collection_string': root_instance.get_collection_string(),
-                'pk': root_instance.pk,
-                'is_deleted': is_deleted and instance == root_instance,
-                'dispatch_uid': root_instance.get_access_permissions().get_dispatch_uid()})
-        except ChannelLayer.ChannelFull:
-            pass
+        # If currently there is an open database transaction, then the following
+        # function is only called, when the transaction is commited. If there
+        # is currently no transaction, then the function is called immediately.
+        def call_channel():
```
Comment (normanjaeckel) [inline openslides/utils/autoupdate.py]: Suggestion: Rename this to `call_autoupdate_channel` or something similar.

**Comment by normanjaeckel** [comment]
Nice. One suggestion to @ostcar.

@ThomasJunk Can you please test if this fix works for you?

**Comment by ThomasJunk** [comment]
Does the trick :+1:

**Comment by emanuelschuetze** [comment]
I tested successfully this pull request:
After creating new motion there are no `openslides.motions.models.DoesNotExist` any more as reported in #2248. 

@ostcar Thanks. Please look at Norman's suggestion. Then we can merge it.

File: openslides/utils/autoupdate.py (hunk 1 above)
Comment (ostcar) [inline openslides/utils/autoupdate.py]: done (with other name)

**Comment by normanjaeckel** [comment]
Please rebase.

**Pull request merged**